content/docs/instrumenting/exporters.md

```diff
@@ -25,3 +26,23 @@ following is a list of existing third-party exporters:
 The [JMX exporter](https://github.com/prometheus/jmx_exporter) can export from a
 wide variety of JVM-based applications, for example [Kafka](http://kafka.apache.org/) and
 [Cassandra](http://cassandra.apache.org/).
+
+Unofficial third-party exporters:
+
```

Comment:
I added something to that effect, and changed the text/headline structure a bit.